classroom.rb

```diff
@@ -0,0 +1,14 @@
+# Class representing a classroom.
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student
```

Comment:
- [ ] Kindly make sure the student (self) is added to the classroom variable from the student class like so `student.classroom = self`